### DISCUSSION TITLE
can i use date object in day picker at modifier?

### DISCUSSION THREAD

**Discussion by chlgbdud** [gpbl/react-day-picker#2034] [Support]
i made array,

[{day: 0, week:[1,2]},{day: 1, week[1,2]}]

i want use date object in onSelect at modifiers props.

i read document, i find DayofWeek matcher. but my array isn't disabled all weeks.

how can i do?

i made function, when date is repeat, return boolen by comparing my array.

**Comment by gpbl** [comment]
Hi @chlgbdud please provide a better description of your issue. What are you trying to do?
